docs/devguide/modules-dev-guide.asciidoc

```diff
@@ -485,8 +485,10 @@ run.
 In order to test the filesets with the sample logs and/or generate the expected output one should run the tests
 locally for a specific module, using the following procedure under Filebeat directory:
 
-. Run an Elasticsearch instance locally using docker: `docker run -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:8.0.0-SNAPSHOT`
+. Run an Elasticsearch instance locally using docker: `docker run --rm --name elasticsearch -p 9200:9200 -p 9300:9300 -e "ELASTIC_PASSWORD=changeme" -e "discovery.type=single-node" -e "xpack.security.http.ssl.enabled=false" docker.elastic.co/elasticsearch/elasticsearch:8.1.0-SNAPSHOT`
```

Comment:
I think the idea of using `8.0.0-SNAPSHOT` was to use the snapshot from master. Do you know if there is now any other snapshot following master? Otherwise we may need to update this on every minor release :thinking:

Reply:
Looks like `master-SNAPSHOT` works. Presumably this will switch to `main-SNAPSHOT` at some point but we can update when that happens and it should only happen once.